hutch_python/ipython_log.py

```diff
@@ -6,13 +6,34 @@
 import functools
 import logging
 import sys
+import textwrap
+import threading
 import traceback
 
 from .constants import INPUT_LEVEL
 from .log_setup import log_exception_to_central_server
 
 logger = logging.getLogger(__name__)
 logger.input = functools.partial(logger.log, INPUT_LEVEL)
+logger.propagate = False
```

Comment:
Can you explain what turning off propagate does in this context? Does this not prevent the input logs from reaching the logger file?

Reply:
Yeah, this is almost certainly in error, now that you mention it.
Thanks for pointing it out, or we wouldn't have caught it until we tested this on a hutch machine 👍

Reply:
Reflecting on this last night, this was because - without going into too much detail - I did some stupid things while testing this interactively. I think I have more to fix up here, oops. 😁

Reply:
Resolved in de2862d